frontend/javascripts/oxalis/controller/scene_controller.js

```diff
@@ -251,6 +251,11 @@ class SceneController {
       showCrossSections: true,
     });
     this.userBoundingBox.getMeshes().forEach(mesh => this.rootNode.add(mesh));
+    const { viewMode } = Store.getState().temporaryConfiguration;
+    if (viewMode === "flight") {
+      // We disable the bounding box in flight mode as it distracts the user.
+      this.userBoundingBox.setVisibility(false);
```

Comment:
Does this need to be undone when switching to ortho mode? Or is the scene controller re-initialized in that case, anyway?

Reply:
* When ortho mode is loaded `startPlaneMode` is called. There all bounding boxes are set visible again.
* When the user changes to any arbitrary mode the `stopPlaneMode`-method is called that makes all bounding boxes invisible.

I hope this answers your question :D

Reply:
Hm, then I don't understand why it's necessary to set the visibility to false here, at all? At the end of `createMeshes` `this.stopPlaneMode();` is called which should hide all boxes, anyway?

Reply:
Ok, your are completely correct here. I missed that :see_no_evil: . Thanks for noticing

Reply:
But in `buildTaskingBoundingBox` we still need to eventually disable the task bounding box, as this method is also called each time the task bounding box is changed in the store.

Or are you confident that the task bounding box will never change in the store?

Reply:
> But in buildTaskingBoundingBox we still need to eventually disable the task bounding box, as this method is also called each time the task bounding box is changed in the store.

Yes, you are right, the code which reacts to store changes should set the correct visibility :+1: